test-apps/display-test-app/src/common/DtaRpcInterface.ts

```diff
@@ -30,4 +30,5 @@ export class DtaRpcInterface extends RpcInterface {
   public async writeExternalFile(_filename: string, _content: string): Promise<void> { return this.forward(arguments); }
   public async terminate(): Promise<void> { return this.forward(arguments); }
   public async getEnvConfig(): Promise<DtaConfiguration> { return this.forward(arguments); }
+  public async getAccessToken(): Promise<string> { return this.forward(arguments); }
```

Comment:
Why are we sending a token via RPC?

Reply:
That's how dpta [did it](https://github.com/iTwin/itwinjs-core/blob/31b44c26417653dec338d6550cb0bd61fc75a0a1/test-apps/display-performance-test-app/src/common/DisplayPerfRpcInterface.ts#L50) in their [PR](https://github.com/iTwin/itwinjs-core/pull/3822).